src/Goerli-DssSpell.sol

```diff
@@ -23,22 +23,75 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
+import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
+import { JugAbstract } from "dss-interfaces/dss/JugAbstract.sol";
+import { IlkRegistryAbstract } from "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import { DaiAbstract } from "dss-interfaces/dss/DaiAbstract.sol";
+
 interface RwaUrnLike {
     function draw(uint256) external;
 }
 
+interface CureLike {
+    function lift(address) external;
+}
+
+interface TeleportJoinLike {
+    function rely(address) external;
+    function file(bytes32,address) external;
+    function file(bytes32,bytes32,address) external;
+    function file(bytes32,bytes32,uint256) external;
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function domain() external view returns (bytes32);
+}
+
+interface TeleportRouterLike {
+    function rely(address) external;
+    function file(bytes32,bytes32,address) external;
+    function gateways(bytes32) external view returns (address);
+    function domains(address) external view returns (bytes32);
+    function dai() external view returns (address);
+}
+
+interface TeleportOracleAuthLike {
+    function rely(address) external;
+    function file(bytes32,uint256) external;
+    function addSigners(address[] calldata) external;
+    function teleportJoin() external view returns (address);
+}
+
+interface EscrowLike {
+    function approve(address,address,uint256) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
-    uint256 constant internal MILLION  = 10 **  6;
-    uint256 constant internal BILLION  = 10 **  9;
-    uint256 constant internal WAD      = 10 ** 18;
+    address internal constant TELEPORT_JOIN = 0xE2fddf4e0f5A4B6d0Cc1D162FBFbEF7B6c5D6f69;
+    address internal constant ORACLE_AUTH = 0x29d292E0773E484dbcA8626F432985630175763b;
+    address internal constant ROUTER = 0x5A16311D32662E71f1E0beAD41372f60cEb61b26;
+    address internal constant LINEAR_FEE = 0x89bcDc64090ddAbB9AFBeeFB7999d564e2875907;
+
+    bytes32 internal constant ILK = "TELEPORT-FW-A";
+    bytes32 internal constant DOMAIN_ETH = "ETH-GOER-A";
 
-    // address constant RWA008_A_OPERATOR       = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
-    // address constant RWA008_A_OUTPUT_CONDUIT = 0x1aA21d2E39EC0da185CA04609c8868bC324d8553;
-    uint256 constant RWA009_DRAW_AMOUNT      = 25_000_000 * WAD;
+    bytes32 internal constant DOMAIN_OPT = "OPT-GOER-A";
+    address internal constant TELEPORT_GATEWAY_OPT = 0x5d49a6BCEc49072D1612cA6d60c8D7985cfc4988;
```

Comment:
Not big deal if preferred this way, but IMO is easier to just name the variables as the chainlog key, uses to be faster for verifying them with other sources.

Reply:
Going to just leave it if that's cool.

Reply:
Sure! resolving